modules/utxo-lib/src/bitgo/Musig2.ts

```diff
@@ -0,0 +1,214 @@
+import { PSBT_PROPRIETARY_IDENTIFIER, PsbtProprietaryKeyValueData, UtxoPsbt } from './UtxoPsbt';
+import { checkPlainPublicKey, checkXOnlyPublicKey, toXOnlyPublicKey } from './outputScripts';
+import { BIP32Interface } from 'bip32';
+import { ecc, musig } from '../noble_ecc';
+import { Tuple } from './types';
+import { tapTweakPubkey } from '../taproot';
+import { TapBip32Derivation } from 'bip174/src/lib/interfaces';
+
+export const PSBT_IN_MUSIG2_PARTICIPANT_PUB_KEYS_SUBTYPE = 0x01;
```

Comment:
Let's move the enum currently living in `ZcashPsbt` up to `UtxoPsbt` and use that instead.

Reply:
Done